controllers/util/secrets_util.go

```diff
@@ -230,7 +281,28 @@ func newS3ConfigurationSecret(s3SecretRef corev1.SecretReference, targetns strin
 	}
 }
 
-func newConfigurationPolicy(name string, object runtime.RawExtension) *cpcv1.ConfigurationPolicy {
+func newVeleroSecret(s3SecretRef corev1.SecretReference, fromNS, veleroNS, keyName string) *localSecret {
+	return &localSecret{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Secret",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "v" + s3SecretRef.Name, // Disambiguate with a "v" in case fromNS and veleroNS are the same
+			Namespace: veleroNS,
+		},
+		Data: map[string]string{
+			keyName: "{{ (printf \"default\\n  aws_access_key_id = %s\\n  aws_secret_access_key = %s\" " +
+				"((lookup \"v1\" \"Secret\" \"" + fromNS +
+				"\" \"" + s3SecretRef.Name + "\").data.AWS_ACCESS_KEY_ID | base64dec) " +
+				"((lookup \"v1\" \"Secret\" \"" + fromNS +
+				"\" \"" + s3SecretRef.Name + "\").data.AWS_SECRET_ACCESS_KEY | base64dec)" +
+				") | base64enc }}",
```

Comment:
This policy syntax is very hard to work with. Can we avoid it?

Or at lest prepare this as a multiline string to we can get everything lined up nicely and avoid the extra quoting?

Reply:
Added a comment with the desired output that it should generate